src/ElementUtilities.js

```diff
@@ -21,7 +21,7 @@ const findClosest = (el, selector) => {
  * @return {boolean}        Whether table ancestor of 'el' is found
```

Comment:
Please also update findClosest and other methods to eliminate parentheses usage.